internal/persistence/sql/migrations/templates/20210623162418_namespace_id.up.fizz

```diff
@@ -0,0 +1,9 @@
+create_table("keto_namespace_ids") {
+    t.Column("nid", "uuid", {})
+    t.Column("id", "uuid", {})
+    t.Column("serial_id", "integer", {})
+
+    t.PrimaryKey("nid", "id")
+}
+
+add_index("keto_namespace_ids", ["nid", "serial_id"], {unique: true})
```

Comment:
Are we looking up by serial_id? If so please add an index

Reply:
This is already the index for that, as we always only look within one network ID.